### PR TITLE
Add info for large customer azure ad sign in logs

### PR DESCRIPTION
**Pull request by grtaylor806**
@chrisda

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/deprecation-of-basic-authentication-exchange-online.md [hunk 1]
```diff
@@ -156,6 +156,12 @@ Early in 2022, we plan on updating the Microsoft Admin Center to make it easier
 
 The best place to get the most up-to-date picture of Basic authentication usage by tenants is by using the Azure AD Sign-In report. To learn more, see: [New tools to block legacy authentication in your organization - Microsoft Tech Community](https://techcommunity.microsoft.com/t5/azure-active-directory-identity/new-tools-to-block-legacy-authentication-in-your-organization/ba-p/1225302). 
 
+If you want to export logs for analysis it does require a premium license for your Azure AD tenant. If you have that you can use the following methods to export logs;
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/deprecation-of-basic-authentication-exchange-online.md:159]:
```suggestion
If you want to export logs for analysis, you require a premium license for your Azure AD tenant. If you have a premium license you can use the following methods to export logs:
```

File: Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/deprecation-of-basic-authentication-exchange-online.md [hunk 2]
```diff
@@ -156,6 +156,12 @@ Early in 2022, we plan on updating the Microsoft Admin Center to make it easier
...
+
+- In general, we recommend customers prioritize data export to event hubs, Azure Storage, or Azure Monitor. Those export pathways are all capable of handling the load from even customer tenants with hundreds of thousands of users. [Stream Azure Active Directory logs to Azure Monitor logs](/azure/active-directory/reports-monitoring/howto-integrate-activity-logs-with-log-analytics)
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/deprecation-of-basic-authentication-exchange-online.md:161]:
```suggestion
- We recommend customers prioritize data export to event hubs, Azure Storage, or Azure Monitor. Those export pathways are all capable of handling the load from even customer tenants with hundreds of thousands of users. For more information, see [Stream Azure Active Directory logs to Azure Monitor logs](/azure/active-directory/reports-monitoring/howto-integrate-activity-logs-with-log-analytics).
```

File: Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/deprecation-of-basic-authentication-exchange-online.md [hunk 3]
```diff
@@ -156,6 +156,12 @@ Early in 2022, we plan on updating the Microsoft Admin Center to make it easier
...
+-	You can also use Graph APIs to export sign in logs, but we recommend you implement MS Graph paging logic to ensure you can pull all the logs. [Access Azure AD logs with the Microsoft Graph API](azure/active-directory/reports-monitoring/quickstart-access-log-with-graph-api)
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/deprecation-of-basic-authentication-exchange-online.md:162]:
```suggestion
- You can also use Graph APIs to export sign-in logs, but we recommend you implement MS Graph paging logic to ensure you can pull in all the logs. For more information, see [Access Azure AD logs with the Microsoft Graph API](azure/active-directory/reports-monitoring/quickstart-access-log-with-graph-api).
```

File: Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/deprecation-of-basic-authentication-exchange-online.md [hunk 4]
```diff
@@ -156,6 +156,12 @@ Early in 2022, we plan on updating the Microsoft Admin Center to make it easier
...
 
+If you want to export logs for analysis it does require a premium license for your Azure AD tenant. If you have that you can use the following methods to export logs;
+
+- In general, we recommend customers prioritize data export to event hubs, Azure Storage, or Azure Monitor. Those export pathways are all capable of handling the load from even customer tenants with hundreds of thousands of users. [Stream Azure Active Directory logs to Azure Monitor logs](/azure/active-directory/reports-monitoring/howto-integrate-activity-logs-with-log-analytics)
+-	You can also use Graph APIs to export sign in logs, but we recommend you implement MS Graph paging logic to ensure you can pull all the logs. [Access Azure AD logs with the Microsoft Graph API](azure/active-directory/reports-monitoring/quickstart-access-log-with-graph-api) 
+- You can download logs directly from yoour browser, but for large customers the amount of data can cause browser timeouts.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/clients-and-mobile-in-exchange-online/deprecation-of-basic-authentication-exchange-online.md:163]:
```suggestion
- Finally, you can download logs directly from your browser, but for large customers, the amount of data can cause browser timeouts. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Pull request merged**